test/acceptance/features/bindAppToServiceAnnotations.feature

```diff
@@ -0,0 +1,301 @@
+@annotations
+Feature: Bind an application to a service using annotations
+
+    As a user of Service Binding Operator
+    I want to bind applications to services it depends on
```

Comment:
Please be here more specific, given the annotation scope of the new added feature

Reply:
done